src/internal/operations/restore_test.go

```diff
@@ -218,38 +222,68 @@ func (suite *RestoreOpIntegrationSuite) SetupSuite() {
 	require.NotEmpty(t, bo.Results.BackupID)
```

Comment:
nit: can you actually make separate subtests for each backup type? That will help with output in github actions. It's hard to find the output for the service we want if the setup fails because it has both assertions and log output all under one section and some services are chatty with the logs

Reply:
Oh, good idea!  Can do.